ONSdigital/eq-survey-runner
# [eq-769] UKIS NI version 1

<img width="1086" alt="screenshot 2019-01-09 11 42 48" src="https://user-images.githubusercontent.com/930398/50897338-bbd0ee00-1403-11e9-9132-0ac4d99b3566.png">

### What is the context of this PR?

[Trello card](https://trello.com/c/d0lBPK9k/769-ukis-ni-version-1)

- Adds UKIS Northern Ireland theme
- Adds UKIS Ni schema
- [Tweaks to existing UKIS theme logo](https://trello.com/c/3bjw7DSR/786-ukis-logo-re-size)

**NOTE: I had to move the the grid column out of `partials/services-navigation` (it shouldn't be in a component anyway, this breaks encapsulation) and make it a concern of the header template so will require additional testing** 

### How to review 

1. Start survey using `ukis_0002.json` schema.
2. Check correct survey and correct logos appear in header.
3. Confirm changes made elsewhere (to `partials/services-navigation`) haven't broken any other affected themes.
